source/Contribution-Guide/addingpages.html.md.erb

```diff
@@ -28,21 +30,28 @@ The first page in every folder should be called **index.html.md.erb** and start
     
     # <%%= current_page.data.title %>
     
+The weight parameter controls the pages position in the master index, with lower numbers appearing at the top. Increment the **count by 10** where possible
+
 
-**Weight** is used to control the pages position in the master index.  With low numbers at the top.  Where possible **increment the count by 10**. 
+## Best Practices for Well-Written Pages
 
-## Well Written Pages
+**Structure**
 
-- Prerequisite Section at top
-- Diagrams (A good diagram, tells the story)
-- Admin Tools
+- Include a Prerequisites section at the top
+- Use diagrams to illustrate effectively (A good diagram, tells the story)
+- Provide information on Admin Tools:
     - Admin portal (URL or IP address)
-    - Instructions to download, install and configure
+    - Instructions for downloading, installing, and configuring
     - Describe with text, how to navigate within the system
-- Follow house style guide
-    - Use empty template for new products and applications
-    - When section is empty, leave it in place but add the word blank to show it is not relevant
-- Core sections
-    - Introduction or Description (What is it)
-    - Prerequisites (Can it be made to work by reader)
-    - Location (Links, URLs, IP addresses or pointers)
\ No newline at end of file
+
+**Style Guide**
+
+- Follow the house style guide and check out [guide examples](guide-examples/index.html) to assist you in creating your own guides
+- Use an empty template for new products and applications
+- When section is empty, leave it in place but add the word blank to show it is not relevant 
+
+**Core Sections**
+
+1. Introduction or Description (What is it)
+2. Prerequisites (Can the reader make it work)
+3. Location (Links, URLs, IP addresses, or pointers)
```

Comment:
Replace `location` with `Facts`, covers any information related to the thing

Reply:
ty